platform/security/certificate/security-certificate-keystoreeditor/pom.xml

```diff
@@ -73,7 +85,9 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Embed-Dependency>
-                            not-yet-commons-ssl
+                            not-yet-commons-ssl,
```

Comment:
Is this project still alive?  It has not been updated since 2009.  

I also do not see it in the dependency list above.  Did you remove it and forget to remove this line?